webapp/app/forms/steps/unlock_code_revocation_steps.py

```diff
@@ -13,8 +13,6 @@
 from app.model.components.helpers import form_fields_dict
 from app.templates.react_template import render_react_template
 
-HEADER_TITLE = 'form.unlock-code-revocation.header-title'
```

Comment:
This will not work because the babel script generation search for uses of `_()` or `_l()`...